int64 MissionStatus
---
string FeedbackMsg
